.compute

```diff
@@ -4,7 +4,7 @@
 # in our case we use the provided COMPUTE_* env variables
 # to construct a distributed TensorFlow cluster definition
 
-set -e
+set -o pipefail
```

Comment:
This is one of the most important lessons of this change. `set -e` will NOT cause the script to immediately fail and exit, if the failing process is not the last part of a pipe cascade. `set -o pipefail` fixes that. 
So why no `-e` to exit due to now failing pipe cascades? Because I want to capture status codes and print a graceful exit note.